normandy/recipes/signing.py

```diff
@@ -190,9 +191,15 @@ def verify_x5u(url, expire_early=None):
     chain. Otherwise, raise an exception explaining why they are not
     valid.
     """
-    req = requests.get(url)
-    req.raise_for_status()
-    pem = req.content.decode()
+
+    cache_key = f"fetch-x5u-pem::{url}"
+
+    pem = cache.get(cache_key)
+    if pem is None:
+        req = requests.get(url, timeout=10)
```

Comment:
Isn't the returned object a response? (`resp = requests.get()`?)

Also, you don't want the timeout as constant or a setting?

Reply:
Ah, good point.

I thought about making the timeout a setting, but I don't think it's really going to change. A constant would be a good idea.